merch-dropper/src/components/FAQ.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import {ExpansionPanel, ExpansionPanelSummary, ExpansionPanelDetails, Typography} from '@material-ui/core';
+
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: '70%',
+    height: 500,
+    margin: "auto",
+  },
+  heading: {
+    fontSize: theme.typography.pxToRem(18),
+    fontWeight: theme.typography.fontWeightRegular,
+  },
+  panel: {
+    borderBottom: "1px solid F3F3F3"
+  },
+  
+
+}));
+
+const FAQ = () =>{
+  const classes = useStyles();
+
+  return (
+    <div className={classes.root}>
+        <h1 style={{textAlign: "center", marginBottom: 50}}>Frequently Asked Questions</h1>
+      <ExpansionPanel className={classes.expansion}>
+      <ExpansionPanelSummary
+        className={classes.panel}
+          expandIcon={<ExpandMoreIcon />}>
+          <Typography className={classes.heading}>What is Stripe?</Typography>
+        </ExpansionPanelSummary>
+        <ExpansionPanelDetails>
+          <Typography>
+          Stripe is a software that allows stores to accept payments online.  There is a 2.9% + $0.30 fee from stripe for each transaction.
+          </Typography>
+        </ExpansionPanelDetails>
+      </ExpansionPanel>
+      <ExpansionPanel className={classes.expansion}>
+        <ExpansionPanelSummary
+        className={classes.panel}
+          expandIcon={<ExpandMoreIcon />}>
+          <Typography className={classes.heading}>How do I add products to my store?</Typography>
+        </ExpansionPanelSummary>
+        <ExpansionPanelDetails>
+          <Typography>
+            From your dashboard, you will click on "Add Product".  From there you will upload the design that you would like to have on your merchandise, and select a color.  Then click on preview design.  If you are satisfied, click on Save & Continue.  This will take you to a page where you can name your item, give it a price, and set a description.  It is as easy at that.
+          </Typography>
+        </ExpansionPanelDetails>
+      </ExpansionPanel >
+      <ExpansionPanel className={classes.expansion}>
+      <ExpansionPanelSummary
+      className={classes.panel}
+          expandIcon={<ExpandMoreIcon />}>
+          <Typography className={classes.heading}>How do I share my store with others?</Typography>
+        </ExpansionPanelSummary>
+        <ExpansionPanelDetails>
+          <Typography>
+            When you create a store, you are able to chose a personalized name for your store.  You can share the address with anyone that you would like to purchase from your website.  The link will be merchdropper.store/"YourStore"
```

Comment:
When you create a store you are given a personalized domain name (merchdropper.store/your_store) that will be the storefront for your designed products.  Share your new url address with friends, family, and followers! Copy and paste, text and even email at your heart's content; we don't judge.

Reply:
Thank you